macros/src/actix/mod.rs

```diff
@@ -90,6 +90,60 @@ pub fn emit_v2_operation(input: TokenStream) -> TokenStream {
     .into()
 }
 
+/// Actual parser and emitter for `api_v2_errors` macro.
+pub fn emit_v2_errors(attrs: TokenStream, input: TokenStream) -> TokenStream {
+    let item_ast = match crate::expect_struct_or_enum(input) {
+        Ok(i) => i,
+        Err(ts) => return ts,
+    };
+
+    let name = &item_ast.ident;
+    let attrs = crate::parse_input_attrs(attrs);
+    let generics = item_ast.generics.clone();
+    let (impl_generics, ty_generics, where_clause) = generics.split_for_impl();
+
+    // Convert macro attributes to tuples in form of ("error_code", "error_message")
+    let error_codes = attrs
+        .0
+        .iter()
+        .filter_map(|attr| match attr {
+            NestedMeta::Lit(Lit::Str(attr)) => {
+                let attr_value = attr.value();
+                let mut chunks = attr_value.splitn(2, ' ').map(str::to_string);
+
+                let error_code = chunks.next().unwrap_or_else(|| "".to_string());
+                let error_message = chunks.next().unwrap_or_else(|| "".to_string());
+
```

Comment:
We have to ensure that the status code is valid (or throw error at compile time), given that we're should generate a valid OpenAPI spec.